be/src/exec/vectorized/aggregator.cpp

```diff
@@ -723,8 +748,37 @@ void Aggregator::_init_agg_hash_variant(HashVariantType& hash_variant) {
         }
         }
     }
+
+    int real_fixed_size = 0;
+    if (_group_by_expr_ctxs.size() > 1) {
```

Comment:
why > 1

Reply:
because we are optimizing multi-column groupby cases.

Reply:
nice question. maybe this optimization can be applied to a single column too.